### ISSUE TITLE
Possible to combine both extension

### ISSUE THREAD

**Issue by aayushrautela** [marcinjahn/gnome-quicksettings-audio-devices-renamer-extension#2]
Since most people want to remove and rename together, why not just merge them?

**Comment by marcinjahn** [comment]
Well, I haven’t run or seen any surveys on the topic, so I cannot be sure if your statement is correct :) More seriously, I organized the projects this way, because I like when an extension focuses on one thing. Since it was possible to separate the rename and hide functionalities into two things, I did.

On a side note, the names of these extensions already are a bit too verbose, I don’t know what I’d call the merged one :)

is there any particular reason why you’d want these two extensions to merge into one?

**Comment by aayushrautela** [comment]
Understandable. My only point was to declutter the extensions app since the features are closed linked to each other. Maybe 'qs panel audio manager'?

**Comment by marcinjahn** [comment]
The thing worth pointing out is that search on [EGO](https://extensions.gnome.org/) is bad. No, sorry, it's reaaally bad. Often finding exactly what you're looking for there is surprisingly difficult. This is also the reason why names of my extensions are so explicit. I'm not sure if moving the "hide audio"/"rename audio" bits to description would be a good thing for discoverability, I'm not sure.

Anyway, I'm not planning to merge the two extensions for two reasons:

- it'll be a bit of work to do :slightly_smiling_face: 
- there already are existing users of both extensions. I guess many people would not realize that the extension was migrated to a new one. I'd probably have to add some notifications or whatever, not very convenient.

Still, thanks for asking the question, it's a valid one :slightly_smiling_face:

**Comment by aayushrautela** [comment]
Agreed, the search is horrible there. Well still lots of thanks for maintaining the extensions.
